app/renderer/api/materialize.js

```diff
@@ -34,7 +34,7 @@ module.exports = {
         $('ul.tabs').tabs();
     },
     updateDropdown() {
-        $('.navbar-menu-button').dropdown({
+        $('#navBarMenu').dropdown({
```

Comment:
html classes and ids should be separated by hyphen by convention:
e.g. `navbar-menu`